### PR TITLE
[tflite] do not create unused idle thread when SetNumThreads(1)

### PR DESCRIPTION
**Pull request by mzakharocsc**
…avoid creating unused idle thread when user specifies `interpreter->SetNumThreads(1);`

### CODE REVIEW

**Comment by mzakharocsc** [comment]
I signed it!

File: tensorflow/contrib/lite/kernels/eigen_support.cc [hunk 1]
```diff
@@ -68,7 +68,7 @@ RefCountedEigenContext* GetEigenContext(TfLiteContext* context) {
 void InitDevice(TfLiteContext* context, RefCountedEigenContext* ptr) {
   int num_threads = 4;
   if (context->recommended_num_threads != -1) {
-    num_threads = context->recommended_num_threads;
+    num_threads = (context->recommended_num_threads == 1) ? 0 : context->recommended_num_threads;
```
Comment (aaroey) [inline tensorflow/contrib/lite/kernels/eigen_support.cc]: This exceeds the 80 characters limit, please fix. Same below.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aaroey: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request closed without merging**